docs/additional_reading/heroku_deployment.md

```diff
@@ -3,7 +3,9 @@ The generator has created the necessary files and gems for deployment to Heroku.
 
 + `Procfile`: used by Heroku and Foreman to start the server
 + `.buildpacks`: used to install Ruby and Node environments
++ `config/unicorn.rb`: Unicorn config file
```

Comment:
@justin808 @dylangrafmyre I was told to model the generators off of the tutorial, so that's what I did. I can switch it to Puma, no problem (I switched from Unicorn to Puma on my own personal Rails app a long time ago).